experimental/PropertyDDS/packages/property-dds/src/test/propertyTree.spec.ts

```diff
@@ -111,6 +111,23 @@ describe("PropertyTree", () => {
 				expect((sharedPropertyTree2.root.get("test") as StringProperty).getValue()).to.equal("Magic");
 			});
 
+            it("Can commit with metadata", async () => {
```

Comment:
a test to make sure commit metadata is correctly handled on the committing and receiving end